glide.lock

```diff
@@ -719,7 +719,7 @@ imports:
 - name: github.com/Microsoft/hcsshim
   version: 6ea7fe54f719d95721e7d9b26ac0add224c9b923
 - name: github.com/miekg/dns
-  version: eac804ceef194db2da6ee80c728d7658c8c805ff
+  version: e57bf427e68187a27e22adceac868350d7a7079b
```

Comment:
@deads2k why this bumps?

Reply:
> @deads2k why this bumps?

a couple months ago jordan made it float on a version: https://github.com/openshift/origin/blame/master/glide.yaml#L179  Seems ok to me.  We'll run the pin script when we're done